variables.tf

```diff
@@ -26,49 +26,57 @@ variable ssh_key_name {
 
 variable zone_id {
   description = "Route53 Zone ID"
-  default = ""
+  default     = ""
 }
 
 variable host_name {
   description = "DNS host name"
-  default = ""
+  default     = ""
 }
 
 variable quay_username {
   description = "Quay username"
-  default = "dontspamus"
+  default     = "dontspamus"
 }
 
 variable quay_password {
   description = "Quay password"
 }
 
 variable init_script {
-  description = "bash code executed before `docker-compose up -d` is called, example: `\"${file(\"init.sh\")}\"`"
-  default = ""
+  description = "Script executed before `docker-compose up -d` is called. Example: file(\"init.sh\")"
+  # See how hashicorp doesn't live in reality: https://github.com/hashicorp/terraform/issues/16380
+  default = "echo Setup Completed"
 }
 
 variable in_open_ports {
-  type = "list"
+  type    = list
   default = []
 }
 
 variable in_cidr_blocks {
-  type = "list"
+  type    = list
   default = ["0.0.0.0/0"]
 }
 
 variable in_source_security_group {
   description = "Security group to receive SSH access"
-  type = "string"
-  default = ""
+  type        = string
+  default     = ""
 }
 
 variable disk_utilization_alarm_threshold {
-  description = "disk occupation alarm threshold (% of disk utilization)"
-  default = "80"
+  description = "Disk occupation alarm threshold (% of disk utilization), for example 80.\nIf not set, the alarm won't be created"
+  type        = number
+  default     = 0
+}
+
+variable "sns_topic_arn" {
+  type    = string
+  default = "arn:aws:sns:eu-west-1:309416224681:bellosguardo"
 }
```

Comment:
The naming (and slightly the scope of the variable) could be changed to show that this variable takes care to configure the actions that take place when an alarm is triggered.